### PR TITLE
correct illegal naming of tf.summary.scalar

### PR DESCRIPTION
**Pull request by robrao**
Made the fix for the issue listed below.

`INFO:tensorflow:Summary name learning rate is illegal; using learning_rate instead.`

### CODE REVIEW

**Comment by robrao** [comment]
I signed it!

**Comment by onecityuni** [comment]
I signed it!
